pilot/pkg/model/push_context.go

```diff
@@ -71,10 +71,14 @@ type PushContext struct {
 	// Service related
 	// TODO: move to a sub struct
 
-	// privateServices are reachable within the same namespace.
+	// privateServices are reachable within the same namespace, with exportTo .
```

Comment:
nit: can you please put `.` ->  ``.`` so that we do not get confused with end of sentence?